activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Fix an issue where `ActiveRecord::Encryption::Errors::Encoding` raises when skipping encryption of binary
+    encoded values when querying for unencrypted data.
+
+    *Joshua Young*
+
```

Comment:
```suggestion
```

(Assuming I didn't miss a backport commit) You should not include CHANGELOG entries for bugs that haven't been released, its only for changes since the last release.

Reply:
Of course, I did it out of habit 😅

Reply:
Removed in [this commit](https://github.com/rails/rails/compare/68c7d99cc4cf6c8d24b8837de8fcc222318b4de7..cc6bbefe856c32ebb2fcbb3e5c9dd78bfd5e7f9e).